src/plugins/data_source_management/public/components/data_source_menu/types.ts

```diff
@@ -40,6 +40,7 @@ export interface DataSourceMenuProps<T = any> {
   uiSettings?: IUiSettingsClient;
   application?: ApplicationStart;
   setMenuMountPoint?: (menuMount: MountPoint | undefined) => void;
+  onChangeDevToolsModalVisible: () => void;
```

Comment:
I suggest to rename `onChangeDevToolsModalVisible` -> `onManageDataSource` to make this interface sounds more generic, ideally data source plugin should not maintain logics of dev_tools plugins.

Reply:
Thanks for the advice, already changed the name~